hack/test-cmd.sh

```diff
@@ -151,7 +151,20 @@ export OPENSHIFT_PROFILE="${WEB_PROFILE-}"
 # Specify the scheme and port for the listen address, but let the IP auto-discover. Set --public-master to localhost, for a stable link to the console.
 echo "[INFO] Create certificates for the OpenShift server to ${MASTER_CONFIG_DIR}"
 # find the same IP that openshift start will bind to.  This allows access from pods that have to talk back to master
-SERVER_HOSTNAME_LIST="${PUBLIC_MASTER_HOST},$(openshift start --print-ip),localhost"
+if [[ -z "${ALL_IP_ADDRESSES-}" ]]; then
+    ALL_IP_ADDRESSES="$(openshift start --print-ip)"
+    SERVER_HOSTNAME_LIST="${PUBLIC_MASTER_HOST},localhost,172.30.0.1"
+            SERVER_HOSTNAME_LIST="${SERVER_HOSTNAME_LIST},kubernetes.default.svc.cluster.local,kubernetes.default.svc,kubernetes.default,kubernetes"
```

Comment:
@deads2k I took the contents of `configure_os_server` from `hack/util.sh` and pasted it into `hack/test-cmd.sh` to see what the diff was in config writing between the utility func and our impl here -- the internal names are always added if you use `configure_os_server` and don't provide your own IPs